src/styles/main.scss

```diff
@@ -0,0 +1,129 @@
+$bodyfont: Roboto, sans-serif;
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
+  display: flex;
+  padding: 0;
+  gap: 57px;
```

Comment:
Setting `gap` on the `body` element is not a common practice and might not work as expected across different browsers. It's better to apply `gap` to specific flex or grid containers.